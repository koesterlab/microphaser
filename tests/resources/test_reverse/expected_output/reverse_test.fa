>a324f17d17b218dR
cGAGCCACACCCATCACTCAACATGTC
>c258647e023393eR
TTCcGAGCCACACCCATCACTCAACAT
>172c41a2e6e2cb7R
TTCTTCcGAGCCACACCCATCACTCAA
>b5f424dc651bf48R
CTTTTCTTCcGAGCCACACCCATCACT
>965e45175515136R
GATCTTTTCTTCcGAGCCACACCCATC
>08add1803a50f62R
TGAGATCTTTTCTTCcGAGCCACACCC
>223bd8fb09288ccR
GACTGAGATCTTTTCTTCcGAGCCACA
>732c0f3ff28e5c0R
AGGGACTGAGATCTTTTCTTCcGAGCC
>58c2ffd43739c73R
GGAAGGGACTGAGATCTTTTCTTCcGA
>742163ac1cebc12R
CcGCTGCCTGCCCACCACACGGCCCTC
>fd764a5345f7473R
CATCcGCTGCCTGCCCACCACACGGCC
>a55d1f2587eae5cR
GTGCATCcGCTGCCTGCCCACCACACG
>ae580a513507433R
CTTGTGCATCcGCTGCCTGCCCACCAC
>946de2980abc10fR
GGCCTTGTGCATCcGCTGCCTGCCCAC
>4d2f7903355cae3R
CAAGGCCTTGTGCATCcGCTGCCTGCC
>8a68b9eec214f8bR
GTCCAAGGCCTTGTGCATCcGCTGCCT
>56239f5beba4a30R
CCTGTCCAAGGCCTTGTGCATCcGCTG
>c8cd1f33758a3fcR
CACCCTGTCCAAGGCCTTGTGCATCcG
>0bf5651d15c846cR
acAgggactggggccgggaccgggaca
>683eeee967ad8d0R
gggacAgggactggggccgggaccggg
>7023ad3880e2d57R
accgggacAgggactggggccgggacc
>6f5b9cce754f8f2R
gggaccgggacAgggactggggccggg
>5302a9716f887d3R
gccgggaccgggacAgggactggggcc
>c78a8dc5274f849R
ggggccgggaccgggacAgggactggg
>abb5a5249057784R
actggggccgggaccgggacAgggact
>8aaf4c7d467b4f4R
gggactggggccgggaccgggacAggg
>18fa98f4b970791R
accgggactggggccgggaccgggacA
>f7a0e246701c5ddR
acAgggactggggccgggaccgggacA
