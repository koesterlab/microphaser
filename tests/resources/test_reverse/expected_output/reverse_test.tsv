id	transcript	gene_id	gene_name	chrom	offset	freq	depth	nvar	nsomatic	nvariant_sites	nsomvariant_sites	strand	variant_sites	somatic_positions	somatic_aa_change	germline_positions	germline_aa_change	normal_sequence	mutant_sequence
a324f17d17b218dR	ENST00000314675	ENSG00000158062	UBXN11	chr1	26300986	0.5229357798165137	109	1	1	1	1	Reverse	26300986	26300986	p.Ser46Ser			TGAGCCACACCCATCACTCAACATGTC	cGAGCCACACCCATCACTCAACATGTC
c258647e023393eR	ENST00000314675	ENSG00000158062	UBXN11	chr1	26300983	0.5137614678899083	109	1	1	1	1	Reverse	26300986	26300986	p.Ser46Ser			TTCTGAGCCACACCCATCACTCAACAT	TTCcGAGCCACACCCATCACTCAACAT
172c41a2e6e2cb7R	ENST00000314675	ENSG00000158062	UBXN11	chr1	26300980	0.5087719298245614	114	1	1	1	1	Reverse	26300986	26300986	p.Ser46Ser			TTCTTCTGAGCCACACCCATCACTCAA	TTCTTCcGAGCCACACCCATCACTCAA
b5f424dc651bf48R	ENST00000314675	ENSG00000158062	UBXN11	chr1	26300977	0.5	120	1	1	1	1	Reverse	26300986	26300986	p.Ser46Ser			CTTTTCTTCTGAGCCACACCCATCACT	CTTTTCTTCcGAGCCACACCCATCACT
965e45175515136R	ENST00000314675	ENSG00000158062	UBXN11	chr1	26300974	0.504	125	1	1	1	1	Reverse	26300986	26300986	p.Ser46Ser			GATCTTTTCTTCTGAGCCACACCCATC	GATCTTTTCTTCcGAGCCACACCCATC
08add1803a50f62R	ENST00000314675	ENSG00000158062	UBXN11	chr1	26300971	0.488	125	1	1	1	1	Reverse	26300986	26300986	p.Ser46Ser			TGAGATCTTTTCTTCTGAGCCACACCC	TGAGATCTTTTCTTCcGAGCCACACCC
223bd8fb09288ccR	ENST00000314675	ENSG00000158062	UBXN11	chr1	26300968	0.47580645161290325	124	1	1	1	1	Reverse	26300986	26300986	p.Ser46Ser			GACTGAGATCTTTTCTTCTGAGCCACA	GACTGAGATCTTTTCTTCcGAGCCACA
732c0f3ff28e5c0R	ENST00000314675	ENSG00000158062	UBXN11	chr1	26300965	0.4796747967479675	123	1	1	1	1	Reverse	26300986	26300986	p.Ser46Ser			AGGGACTGAGATCTTTTCTTCTGAGCC	AGGGACTGAGATCTTTTCTTCcGAGCC
58c2ffd43739c73R	ENST00000314675	ENSG00000158062	UBXN11	chr1	26300962	0.5203252032520326	123	1	1	1	1	Reverse	26300986	26300986	p.Ser46Ser			GGAAGGGACTGAGATCTTTTCTTCTGA	GGAAGGGACTGAGATCTTTTCTTCcGA
742163ac1cebc12R	ENST00000314675	ENSG00000158062	UBXN11	chr1	26284398	1.0	32	1	1	1	1	Reverse	26284399	26284399	p.Leu312Arg			CAGCTGCCTGCCCACCACACGGCCCTC	CcGCTGCCTGCCCACCACACGGCCCTC
fd764a5345f7473R	ENST00000314675	ENSG00000158062	UBXN11	chr1	26284395	1.0	33	1	1	1	1	Reverse	26284399	26284399	p.Leu312Arg			CATCAGCTGCCTGCCCACCACACGGCC	CATCcGCTGCCTGCCCACCACACGGCC
a55d1f2587eae5cR	ENST00000314675	ENSG00000158062	UBXN11	chr1	26284392	1.0	34	1	1	1	1	Reverse	26284399	26284399	p.Leu312Arg			GTGCATCAGCTGCCTGCCCACCACACG	GTGCATCcGCTGCCTGCCCACCACACG
ae580a513507433R	ENST00000314675	ENSG00000158062	UBXN11	chr1	26284389	1.0	33	1	1	1	1	Reverse	26284399	26284399	p.Leu312Arg			CTTGTGCATCAGCTGCCTGCCCACCAC	CTTGTGCATCcGCTGCCTGCCCACCAC
946de2980abc10fR	ENST00000314675	ENSG00000158062	UBXN11	chr1	26284386	1.0	33	1	1	1	1	Reverse	26284399	26284399	p.Leu312Arg			GGCCTTGTGCATCAGCTGCCTGCCCAC	GGCCTTGTGCATCcGCTGCCTGCCCAC
4d2f7903355cae3R	ENST00000314675	ENSG00000158062	UBXN11	chr1	26284383	1.0	33	1	1	1	1	Reverse	26284399	26284399	p.Leu312Arg			CAAGGCCTTGTGCATCAGCTGCCTGCC	CAAGGCCTTGTGCATCcGCTGCCTGCC
8a68b9eec214f8bR	ENST00000314675	ENSG00000158062	UBXN11	chr1	26284380	1.0	33	1	1	1	1	Reverse	26284399	26284399	p.Leu312Arg			GTCCAAGGCCTTGTGCATCAGCTGCCT	GTCCAAGGCCTTGTGCATCcGCTGCCT
56239f5beba4a30R	ENST00000314675	ENSG00000158062	UBXN11	chr1	26284377	1.0	34	1	1	1	1	Reverse	26284399	26284399	p.Leu312Arg			CCTGTCCAAGGCCTTGTGCATCAGCTG	CCTGTCCAAGGCCTTGTGCATCcGCTG
c8cd1f33758a3fcR	ENST00000314675	ENSG00000158062	UBXN11	chr1	26284374	1.0	35	1	1	1	1	Reverse	26284399	26284399	p.Leu312Arg			CACCCTGTCCAAGGCCTTGTGCATCAG	CACCCTGTCCAAGGCCTTGTGCATCcG
0bf5651d15c846cR	ENST00000314675	ENSG00000158062	UBXN11	chr1	26282373	0.14634146341463414	41	2	1	3	1	Reverse	26282375|26282381|26282385	26282375	p.Gly496Cys	26282385	p.Pro491_Gly492del	accgggactggggccgggaccgggaca	acAgggactggggccgggaccgggaca
683eeee967ad8d0R	ENST00000314675	ENSG00000158062	UBXN11	chr1	26282370	0.14634146341463414	41	2	1	3	1	Reverse	26282375|26282381|26282385	26282375	p.Gly496Cys	26282385	p.Pro491_Gly492del	gggaccgggactggggccgggaccggg	gggacAgggactggggccgggaccggg
7023ad3880e2d57R	ENST00000314675	ENSG00000158062	UBXN11	chr1	26282367	0.15	40	2	1	3	1	Reverse	26282375|26282381|26282385	26282375	p.Gly496Cys	26282385	p.Pro491_Gly492del	accgggaccgggactggggccgggacc	accgggacAgggactggggccgggacc
6f5b9cce754f8f2R	ENST00000314675	ENSG00000158062	UBXN11	chr1	26282364	0.16216216216216217	37	2	1	3	1	Reverse	26282375|26282381|26282385	26282375	p.Gly496Cys	26282385	p.Pro491_Gly492del	gggaccgggaccgggactggggccggg	gggaccgggacAgggactggggccggg
5302a9716f887d3R	ENST00000314675	ENSG00000158062	UBXN11	chr1	26282361	0.1111111111111111	27	2	1	3	1	Reverse	26282375|26282381|26282385	26282375	p.Gly496Cys	26282385	p.Pro491_Gly492del	gccgggaccgggaccgggactggggcc	gccgggaccgggacAgggactggggcc
c78a8dc5274f849R	ENST00000314675	ENSG00000158062	UBXN11	chr1	26282358	0.041666666666666664	24	1	1	3	1	Reverse	26282360|26282375|26282381	26282375	p.Gly496Cys			ggggccgggaccgggaccgggactggg	ggggccgggaccgggacAgggactggg
abb5a5249057784R	ENST00000314675	ENSG00000158062	UBXN11	chr1	26282355	0.04	25	1	1	3	1	Reverse	26282360|26282375|26282381	26282375	p.Gly496Cys			actggggccgggaccgggaccgggact	actggggccgggaccgggacAgggact
8aaf4c7d467b4f4R	ENST00000314675	ENSG00000158062	UBXN11	chr1	26282352	0.07407407407407407	27	1	1	2	1	Reverse	26282360|26282375	26282375	p.Gly496Cys			gggactggggccgggaccgggaccggg	gggactggggccgggaccgggacAggg
18fa98f4b970791R	ENST00000314675	ENSG00000158062	UBXN11	chr1	26282349	0.038461538461538464	26	1	1	3	1	Reverse	26282351|26282360|26282375	26282375	p.Gly496Cys			accgggactggggccgggaccgggacc	accgggactggggccgggaccgggacA
f7a0e246701c5ddR	ENST00000314675	ENSG00000158062	UBXN11	chr1	26282349	0.038461538461538464	26	2	1	2	0	Reverse	26282351|26282360	26282360	p.Pro501Ser	26282351	p.Gly504Cys	acAgggactggggccgggaccgggacc	acAgggactggggccgggaccgggacA
